sdk/identity/identity/review/identity.api.md

```diff
@@ -104,6 +104,16 @@ export interface AzureDeveloperCliCredentialOptions extends MultiTenantTokenCred
     tenantId?: string;
 }
 
+// @public (undocumented)
+export class AzurePipelinesServiceConnectionCredential implements TokenCredential {
+    constructor(clientId: string, tenantId: string, serviceConnectionId: string, options?: AzurePipelinesServiceConnectionCredentialOptions);
+    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
```

Comment:
The return type here should be `Promise<AccessToken>` to match the rest of the credentials 👍 

I think how that works usually is they throw if we did not get an access token - we should match that behavior